packages/loot-core/src/server/main.ts

```diff
@@ -1099,17 +1099,25 @@ function handleSyncError(err, acct) {
   };
 }
 
-handlers['accounts-bank-sync'] = async function ({ id }) {
+handlers['accounts-bank-sync'] = async function ({ id, ids }) {
   const [[, userId], [, userKey]] = await asyncStorage.multiGet([
     'user-id',
     'user-key',
   ]);
+
+  if ((id && ids) || (!id && !ids)) {
+    throw new Error('Specify either id or ids, but not both');
+  }
+
   const accounts = await db.runQuery(
-    `SELECT a.*, b.bank_id as bankId FROM accounts a
-         LEFT JOIN banks b ON a.bank = b.id
-         WHERE a.tombstone = 0 AND a.closed = 0 ${id ? 'AND a.id = ?' : ''}
-         ORDER BY a.offbudget, a.sort_order`,
-    id ? [id] : [],
+    `  SELECT a.*, b.bank_id as bankId 
+  FROM accounts a
```

Comment:
Is this indent correct? Why does the string start with space now?

Reply:
I'm fighting with the linter on this one unfortunately, fixed as best as I can now